testcharms/charm-repo/quantal/lxd-profile/lxd-profile.yaml

```diff
@@ -0,0 +1,45 @@
+#name: juju-default
+description: lxd profile for testing, black list items grouped commented out
+config:
+#
+# allowed config
+#
+  security.nesting: "true"
+  security.privileged: "true"
```

Comment:
This one would require a restart, so we should consider how we highlight this :thinking:

Reply:
agreed, a question/comment in the discourse spec?